fxa/oauth.py

```diff
@@ -9,6 +9,7 @@
 from six import string_types
 from six.moves.urllib.parse import urlparse, urlunparse, urlencode, parse_qs
 
+import jwt
```

Comment:
I'm sure you're aware, but noting for completeness - this new dependency needs to be added to the requirements in `setup.py`.

Reply:
Well it's been about 10 years since I did any python so I definitely forgot. Thanks :-)